app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -52,11 +52,21 @@ def available_repositories
         end
       end
 
+      sorted_groups = repos.sort_by do |repo|
+        major, minor = repo[:substitutions][:releasever].nil? ? 1000.1000 : repo[:substitutions][:releasever].split('.').map(&:to_i)
+        major = major == 0 ? 1000 : major
```

Comment:
What's the significance of checking for 0 here instead of nil? check like minor version?